DataFormats/METReco/interface/HcalPhase1FlagLabels.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DataFormats_METReco_HcalPhase1FlagLabels_h
+#define DataFormats_METReco_HcalPhase1FlagLabels_h
+
+#include "DataFormats/METReco/interface/HcalCaloFlagLabels.h"
+
+namespace HcalPhase1FlagLabels
+{
+    enum HBHEStatusFlag {
+        /*  0 */ HBHEHpdHitMultiplicity = HcalCaloFlagLabels::HBHEHpdHitMultiplicity,
+        /* 11 */ HBHEIsolatedNoise = HcalCaloFlagLabels::HBHEIsolatedNoise,
+        /* 12 */ HBHEFlatNoise = HcalCaloFlagLabels::HBHEFlatNoise,
+        /* 13 */ HBHESpikeNoise = HcalCaloFlagLabels::HBHESpikeNoise,
+        /* 15 */ HBHETS4TS5Noise = HcalCaloFlagLabels::HBHETS4TS5Noise,
```

Comment:
if you plan to change meaning/naming of the same enum values, what is your plan to support 2016 HCAL ?
If both 2016 and future phase1 will be handled by HcalPhase1FlagLabels, it's probably OK
(HcalCaloFlagLabels will just become deprecated and eventually go away).

Please clarify.